Core/Pixel.swift

```diff
@@ -94,6 +94,9 @@ public struct PixelParameters {
     public static let emailKeychainAccessType = "access_type"
     public static let emailKeychainError = "error"
     public static let emailKeychainKeychainStatus = "keychain_status"
+    
+    //
+    public static let bookmarkFolderCount = "folder_count"
```

Comment:
I might be tempted to rename this, as it is it might look like we're sending info on peoples actual bookmarks

Reply:
Renamed to `bookmark_error_orphaned_count`